tests/test_sklearn_voting_classifier_converter.py

```diff
@@ -73,6 +75,8 @@ def test_operator_mul(self):
 
```

Comment:
Could you add unit tests that reflect the fix made, that would have failed otherwise?

Reply:
Fixed.